droopler.install

```diff
@@ -1666,3 +1666,107 @@ function droopler_update_8123(&$sandbox) {
     \Drupal::messenger()->addMessage($sandbox['current'] . ' paragraphs processed, ' . $sandbox['changed'] . ' changed.');
   }
 }
+
+/**
+ * Droopler 2.2 Migrate layout settings.
+ */
+function droopler_update_8124(&$sandbox) {
+
+  $entity_ids = [
+    'd_p_tiles',
+    'd_p_side_tiles',
+    'd_p_side_image',
+    'd_p_side_embed',
+    'd_p_side_form',
+  ];
+
+  if (!isset($sandbox['total'])) {
+    $query = \Drupal::entityQuery('paragraph');
+    $group = $query->orConditionGroup();
+
+    foreach ($entity_ids as $entity_id) {
+      $group->condition('type', $entity_id);
+    }
+
```

Comment:
1. w sumie to takie query jest tworzone 2 razy. Można by pewnie utworzyć je przed Ifem, cos w stylu:
![image](https://user-images.githubusercontent.com/37533838/90025438-5ebfad00-dcb6-11ea-9c10-87d6f205c4d0.png)

2. zamiast tego foreacha i robienia orConditionGroup, to chyba mozna by po prostu zrobic `$query->condition('type', $entity_ids, 'IN')`